doc/source/api/alleleAnnotations.rst

```diff
@@ -0,0 +1,93 @@
+
+Variant Annotation API
+!!!!!!!!!!!!!!!!!!!!!!
+
+See `Allele Annotation schema <../schemas/alleleAnnotations.html>`_ for a detailed reference.
```

Comment:
This was probably less confusing before we removed the graph component. Annotation - such as protein sequence change - happens at the allele level.  For consistency I've changed the documentation to refer to the Allele annotation API in #521.